index.js

```diff
@@ -298,5 +295,7 @@ module.exports = {
         unspecified: 'bottomAlphabetical',
       },
     ],
+    'color-function-notation': 'legacy',
+    'alpha-value-notation': 'number',
```

Comment:
Both of these were reset to allow the previous syntax to continue.

When using the newer values `backpack-node-sass` fails to build - the suggestion here is to defer that problem and allow the older values to continue to be used.